.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: rust
+rust:
+  - stable
+  - beta
+  - nightly
+matrix:
+  allow_failures:
+    - rust: nightly
+  fast_finish: true
+before_install:
+  # minifb requires x11 header files
+  - sudo apt-get update
+  - sudo apt-get install libxcursor1
+before_script:
+  - rustup component add rustfmt
+script:
+  - cargo fmt -- --check
```

Comment:
Maybe let's add an issue for adding rustfmt instead of adding it now. The tool isn't perfect yet and we'll need to evaluate whether it is actually helpful and figure out a good configuration that works for our team. 

Once that is removed, we can just go with the default script that Travis provides if the script section is not specified. No point in having this section if it's the same as the default.